python/tvm/_ffi/ndarray.py

```diff
@@ -283,6 +283,12 @@ def asnumpy(self):
         check_call(_LIB.TVMArrayCopyToBytes(self.handle, data, nbytes))
         return np_arr
 
+    def copybytesto(self, target):
+        assert target.flags['C_CONTIGUOUS']
```

Comment:
Is there a constant anywhere for `'C_CONTIGUOUS'`?

Reply:
Function no longer exists